src/domain/ViewModel.ts

```diff
@@ -115,7 +115,7 @@ export class ViewModel<O extends Options = Options> extends EventEmitter<{change
         return result;
     }
 
-    emitChange(changedProps: any): void {
+    emitChange(changedProps?: any): void {
```

Comment:
Oops. Meant to link [here](https://github.com/ibeckermayer/hydrogen-web/blob/5c5c6007047e10cf0dc00c9d9c5be1b77db99eae/src/domain/login/LoginViewModel.ts#L182).

Reply:
The argument passed to the `emitChange` method does nothing as of now but it will serve some purpose if Hydrogen is ported to another platform. So we'd like to have typescript shout if we forget to pass an argument to the method.

For [the call here](https://github.com/ibeckermayer/hydrogen-web/blob/5c5c6007047e10cf0dc00c9d9c5be1b77db99eae/src/domain/login/LoginViewModel.ts#L182), you can do `emitChange("loginViewModels")`.